integrations/SnapEngage/util.js

```diff
@@ -0,0 +1,74 @@
+import each from "@ndhoule/each";
+
+const integrationContext = {
+  name: "snapengage",
+  version: "1.0.0",
+};
+
+const recordingLiveChatEvents = (updateEventNames, standardEventsMap) => {
+  each((val, key) => {
+    val.forEach((standardEvent) => {
```

Comment:
What if standardEventsMap is empty?

Reply:
Added a new function for that case

Reply:
Modified the code.
We are now handling both things in the same function.
Instead of iterating through `standardEventsMap`, we are directly checking for any eventName present in the mapping.